utils/build-script-impl

```diff
@@ -1797,6 +1803,23 @@ for host in "${ALL_HOSTS[@]}"; do
                     "${swift_cmake_options[@]}"
                 )
 
+                if [[ "${BUILD_TOOLCHAIN_ONLY}" ]]; then
```

Comment:
Should the same comment as line 1626 be repeated here?

Reply:
+1